src/Lykke.Job.MarginTrading.ConsistencyChecker/PeriodicalHandlers/ConsistencyCheckHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using Common;
+using Common.Log;
+using Lykke.Job.MarginTrading.ConsistencyChecker.Core.Services;
+
+namespace Lykke.Job.MarginTrading.ConsistencyChecker.PeriodicalHandlers
+{
+    public class ConsistencyCheckHandler : TimerPeriod
+    {
+        private readonly ILog _log;
+        private readonly IMonitorService _monitorService;
+
+        private bool initialCall;
+
+        public ConsistencyCheckHandler(ILog log, IMonitorService monitorService) :
+            base(nameof(ConsistencyCheckHandler), monitorService.MonitorInterval, log)
+        {
+            _log = log;
+            _monitorService = monitorService;
+            initialCall = true;
+        }
+
+        public override async Task Execute()
+        {
+            if (initialCall)
```

Comment:
We can have it but on this test version I removed it and set it waiting for first period instead of running instantly on app startup, if the best approach is running on startup, lets do so.